mkdocs.yml

```diff
@@ -126,6 +126,7 @@ nav:
         - Java: developer-tools/java.md
         - PHP: developer-tools/php.md
         - OpenAPI Generator: developer-tools/openapi-generator.md
+        - Convert OpenAPI to GraphQL: developer-tools/rest-to-graphql-export.md
```

Comment:
I think maybe I would add this as the same page, all graphQL stuff together and from the sections on the right developers can easily see what content there is